### DISCUSSION TITLE
creat multiple team talk on centos

### DISCUSSION THREAD

**Discussion by farshad-fallah** [BearWare/TeamTalk5#1875] [Q&A] [answered]
hello . i have been creatteam talk server on cent os 7 for 2 mounts
.now i want creat multipl team talk on another port like windows for exam 1400
 could you plese help me how i can thanks

**Comment by bear101** [comment, marked as answer]
you need to create separate tt5srv.xml files, so e.g. tt5srv.1.xml runs on TCP/UDP port 10333 and tt5srv.2.xml runs on TCP/UDP ports 10334.

Alternatively you can use the same tt5srv.xml file and use tt5srv options -tcpport and -udpport.

**Reply by farshad-fallah** [reply]
hi hello thank you  very much i succeeded do that and i creat new team talk on port 10334
thanks
